CMakeLists.txt

```diff
@@ -194,9 +194,28 @@ function(mbed_generate_map_file target)
 endfunction()
 
 #
-# Generate executables
+# To validate profile support based on `target` link libaries.
```

Comment:
That's an implementation details.
It should just be `Validate selected application profile.`